controllers/datasciencecluster/datasciencecluster_controller.go

```diff
@@ -349,7 +350,7 @@ func (r *DataScienceClusterReconciler) reconcileSubComponent(ctx context.Context
 			if enabled {
 				saved.Status.Components.ModelRegistry = &status.ModelRegistryStatus{RegistriesNamespace: mr.RegistriesNamespace}
 			} else {
-				saved.Status.Components.ModelRegistry = nil
+				saved.Status.Components.ModelRegistry = &status.ModelRegistryStatus{}
 			}
```

Comment:
why not move this into f`unc (m *ModelRegistry) UpdateStatus(in *status.ComponentsStatus`) ?

Reply:
yea nice catch, just moved it thanks :)